app/components/createContainer/createContainerController.js

```diff
@@ -217,8 +235,23 @@ function ($scope, $state, $stateParams, Config, Info, Container, Image, Volume,
     config.Volumes = volumes;
   }
 
+  function prepareNetworkConfig(config) {
+    var mode = $scope.network.Mode;
+    var container = $scope.network.Container;
+    var containerName = container;
+    if (container && typeof container == 'object') {
+      containerName = container.Names[0]
```

Comment:
I think when you run `grunt lint` you are going to get an error about a missing semicolon here.